R/database.R

```diff
@@ -1346,22 +1396,24 @@ get_ashm_count_matrix = function(regions_bed,
                                  maf_data,
                                  sample_metadata,
                                  use_name_column = FALSE,
-                                 from_indexed_flatfile = FALSE,
-                                 allow_clustered = FALSE){
+                                 from_indexed_flatfile = TRUE){
 
   if(missing(regions_bed)){
     regions_bed = grch37_ashm_regions
   }
   ashm_maf = get_ssm_by_regions(regions_bed = regions_bed,
-                                streamlined = TRUE,
+                                basic_columns = TRUE,
                                 maf_data = maf_data,
                                 use_name_column = use_name_column,
-                                from_indexed_flatfile = from_indexed_flatfile,
-                                allow_clustered = allow_clustered)
+                                from_indexed_flatfile = from_indexed_flatfile)
 
   ashm_counted = ashm_maf %>%
-    group_by(sample_id, region_name) %>%
+    group_by(Tumor_Sample_Barcode, Region_Name) %>%
     tally()
+
+  colnames(ashm_counted)[1] = "sample_id"
```

Comment:
This does not seem to do anything because the next line re-assigns the name of this column

Reply:
Which line are you saying doesn't do anything? 1412-1414?